### PR TITLE
Make sample code runnable, and add a warning about sandbox environment

### PR DESCRIPTION
**Pull request by changbenny**
For https://github.com/ringcentral/ringcentral-web-phone/issues/35

### CODE REVIEW

File: demo/index.js [hunk 1]
```diff
@@ -192,30 +192,31 @@ function app() {
...
+        localStorage.webPhoneextension = extension || '';
+        localStorage.webPhonePassword = password || '';
+        localStorage.webPhoneRemember = true;
+    }
 
     var sdk = new RingCentral.SDK({
         appKey: apikey, //,
         appSecret: apisecret,//localStorage.webPhoneAppSecret,
-        server: RingCentral.SDK.server.sandbox
+        server: RingCentral.SDK.server.production
```
Comment (kirill-konshin) [inline demo/index.js]: Better add a dropdown where user can choose which environment to use.

**Comment by kirill-konshin** [comment]
Please squash your commits into one:

```
git rebase -i head~3
git push -f
```

File: demo/index.js (hunk 1 above)
Comment (changbenny) [inline demo/index.js]: Nice suggestion, I added it.

**Comment by changbenny** [comment]
I squash the commits and add a dropdown, please review it.

**Comment by kirill-konshin** [comment]
Thank you!

**Pull request merged**